RoomMe/app/src/main/java/com/example/trocket/roomme/HomeActivity.java

```diff
@@ -17,7 +23,16 @@
     private ArrayList<User> userList = new ArrayList<User>();
```

Comment:
Again, we need to change to extends Fragment and adjust accordingly. Look at your old pull request (in Closed section) for my lengthy comment on why.